.github/workflows/transformDataToViews.yml

```
@@ -0,0 +1,59 @@
+# https://github.com/nvaccess/transformAddonDataToViews/
+
+name: Transform NVDA addons to views
+
+on:
+  workflow_dispatch:
```

```suggestion
on:
  workflow_dispatch:
```

@feerrenrut - the action can be manually dispatched. I can't test the action because of the following issue: https://github.community/t/workflow-files-only-picked-up-from-master/16129

---

Ah, this makes sense. The work flow only runs on commits to master, and this config is not yet on master either.

---

Yep, I might be able to push a commit here which changes the .yml rules and triggers the workflow, and then reverse the commit.